front_backpackers/src/FormPage/ActivityForm.js

```diff
@@ -63,7 +65,12 @@ const ActivityForm = props => {
           </Field>
         </div> */}
 
-        <form onSubmit={handleSubmit}>
+        <form
+          method="POST"
+          encType="multipart/form-data"
+          action="upload"
+          onSubmit={handleSubmit}
+        >
```

Comment:
Reloading is cheating
google something like "react node asynchronous upload"
(validate the found ressource with me if needed)